README.md

```diff
@@ -54,19 +54,19 @@ you must add the following dependencies:
 composer require symfony/dependency-injection symfony/http-kernel
 ```
 
-Enable the bundle on the `AppKernel` class:
+Register the bundle in the kernel of your application:
```

Comment:
The change was not about having a different class name, but about talking about what we are doing instead of how (which is covered afterwards).